refactoringminer/src/main/java/gr/uom/java/xmi/decomposition/LeafMapping.java

```diff
@@ -28,6 +31,12 @@ public int compareTo(LeafMapping o) {
                 return -Integer.compare(compositeReplacement1.getTotalAdditionallyMatchedStatements(),
                     compositeReplacement2.getTotalAdditionallyMatchedStatements());
             }
+        } else if (concatenationReplacement1 != concatenationReplacement2) {
+            if (concatenationReplacement1 && !concatenationReplacement2) {
+                return 1;
+            } else {
+                return -1;
+            }
         } else {
             double distance1;
```

Comment:
it is a code duplication (lines 41-59) of the code from CompositeStatementObjectMapping (lines 18-36)
could it be extracted into a separate method?